code/SecurityAdmin.php

```diff
@@ -141,11 +141,11 @@ public function providePermissions()
         return [
             "CMS_ACCESS_SecurityAdmin" => [
                 'name' => _t(
-                    CMSMain::class . '.ACCESS',
+                    LeftAndMain::class . '.ACCESS',
```

Comment:
CMSMain doesn't even exist if you're installing silverstripe/admin without silverstripe/cms. LeftAndMain is the obvious choice here.